README.md

```diff
@@ -90,6 +96,41 @@ To accept inbound connections, add the `--listen` flag.
 $ hsd --listen --max-inbound=20
 ```
 
+### Account and Wallet Setup
+HSD uses a hierarchical deterministic wallet structure. The hsw-cli binary is used to interact with the HSD wallet.
+
+Make sure you have a HSD node running.
+
+To view the extended public key and mnemonic details (entropy and phrase information) by:
+
+`$ hsw-cli master`
+
+To view the `default` account details, such as the `accountKey`, by:
+
+` $ hsw-cli account get default`
+
+To make a new wallet in the `default` account, use the `mkwallet` command and provide a wallet id. It is custom, but not required, to name the first wallet id `primary`.
```

Comment:
This statement is technically incorrect. The statement "To make a new wallet in the default account..." implies that accounts have multiple wallets associated with them. The opposite is actually true. A wallet has a master key from which many accounts can be derived.